src/main/java/io/getunleash/repository/ClientFeaturesResponse.java

```diff
@@ -0,0 +1,65 @@
+package io.getunleash.repository;
+
+import io.getunleash.lang.Nullable;
+
+import java.util.Collections;
+
+public final class ClientFeaturesResponse extends FeatureToggleResponse {
```

Comment:
I struggle a bit to understand why we would want this wrapper around "FeatureToggleResponse". Why can't we simply move the segment collection in to the FeatureToggleResponse implementation and avoid this extra layer of inheritance?

Reply:
I followed the same practice (and class name) as in the node sdk.  Nothing prevents us from using a modified version of the FeatureToggleResponse. imo inheritance is better in this situation, but happy to change if you disagree

Reply:
> imo inheritance is better in this situation

why do you prefer inheritance in this situation?

Reply:
It allows me to make use of the previous functionality, taking away the risk of breaking something existing, 
and the response class matches that of the node sdk.  
Nothing. stopping us from only modifying existing classes to achieve the same results, but I chose to inherit